src/wnaf.rs

```diff
@@ -25,6 +24,63 @@ pub(crate) fn wnaf_table<G: Group>(table: &mut Vec<G>, mut base: G, window: usiz
     }
 }
 
+struct LimbBuffer<'a> {
```

Comment:
I know this is private, nevertheless some documentation would be useful.

```suggestion
/// This struct represents a sequence of bytes that can also be viewed as an
/// sequence of `u64`s in little-endian byte order. The values of consecutive
/// `u64`s can be accessed sequentially.
struct LimbBuffer<'a> {
```